### DISCUSSION TITLE
IRC Channel #diffhtml on freenode

### DISCUSSION THREAD

**Discussion by tbranyen** [tbranyen/diffhtml#218] [General]
I've created an IRC channel #diffhtml on freenode | [irc://irc.freenode.net#diffhtml](irc://irc.freenode.net#diffhtml), feel free to join and discuss issues, features, roadmaps, progress, etc. Maybe there's a better platform, but I figure those who want to hack on something interesting or get ideas will find their way.

**Comment by yatesco** [comment]
Have you considered a discord or telegram group? If you prefer IRC then have you considered the Libre version?

Thanks!
